<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>990</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="plot">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>921</width>
      <height>981</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QCustomPlot" name="plot_solutions">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>561</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_x0">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>280</y>
        <width>55</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>x₀</string>
      </property>
      <property name="margin">
       <number>5</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_x0">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>311</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_X">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>359</y>
        <width>55</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>X</string>
      </property>
      <property name="margin">
       <number>5</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_X">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>390</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_y0">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>470</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_y0">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>439</y>
        <width>55</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>y₀</string>
      </property>
      <property name="margin">
       <number>5</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_N">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>550</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_N">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>519</y>
        <width>55</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>N</string>
      </property>
      <property name="margin">
       <number>5</number>
      </property>
     </widget>
     <widget class="QCustomPlot" name="plot_localErrors">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>561</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_ExactSol">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>100</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Exact</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_EulerSol">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>130</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Euler</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_RkSol">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>190</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Runge-Kutta</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_ImpeulerSol">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>160</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Improved Euler</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_RkLe">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>500</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Runge-Kutta</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_EulerLe">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>440</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Euler</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_ImpeulerLe">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>470</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Improved Euler</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_Solutions">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>551</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Solutions</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_Solutions_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>561</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Local Truncation Errors</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_EulerGe">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>750</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Euler</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_Solutions_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>630</y>
        <width>561</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Global Truncation Errors</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QCustomPlot" name="plot_globalErrors">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>630</y>
        <width>561</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_ImpeulerGe">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>780</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Improved Euler</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_RkGe">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>810</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Runge-Kutta</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="plot_button_main">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>600</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Plot</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_1">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>250</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>570</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_3">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>880</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>3</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>checkBox_1</zorder>
     <zorder>checkBox_3</zorder>
     <zorder>checkBox_2</zorder>
     <zorder>plot_globalErrors</zorder>
     <zorder>label_Solutions_3</zorder>
     <zorder>plot_solutions</zorder>
     <zorder>label_x0</zorder>
     <zorder>lineEdit_x0</zorder>
     <zorder>label_X</zorder>
     <zorder>lineEdit_X</zorder>
     <zorder>lineEdit_y0</zorder>
     <zorder>label_y0</zorder>
     <zorder>lineEdit_N</zorder>
     <zorder>label_N</zorder>
     <zorder>plot_localErrors</zorder>
     <zorder>checkBox_ExactSol</zorder>
     <zorder>checkBox_EulerSol</zorder>
     <zorder>checkBox_RkSol</zorder>
     <zorder>checkBox_ImpeulerSol</zorder>
     <zorder>checkBox_RkLe</zorder>
     <zorder>checkBox_EulerLe</zorder>
     <zorder>checkBox_ImpeulerLe</zorder>
     <zorder>label_Solutions</zorder>
     <zorder>label_Solutions_2</zorder>
     <zorder>checkBox_EulerGe</zorder>
     <zorder>checkBox_ImpeulerGe</zorder>
     <zorder>checkBox_RkGe</zorder>
     <zorder>plot_button_main</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QLabel" name="label_Solutions_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>561</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Total Errors (Global Truncation Errors)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_ImpeulerTe">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>160</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Improved Euler</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_EulerTe">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>130</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Euler</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_RkTe">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>190</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Runge-Kutta</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCustomPlot" name="plot_totalErrors">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>561</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_n0">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>290</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_N_2">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>339</y>
        <width>55</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>N</string>
      </property>
      <property name="margin">
       <number>5</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_n0">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>259</y>
        <width>55</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>n₀</string>
      </property>
      <property name="margin">
       <number>5</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_N_2">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>370</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
     <widget class="QCustomPlot" name="plot_totalErrors_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>561</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_RkTe_2">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>530</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Runge-Kutta</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_EulerTe_2">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>470</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Euler</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_Solutions_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>561</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Total Errors (Local Truncation Errors)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_ImpeulerTe_2">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>500</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Improved Euler</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="plot_button4_3">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>420</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Plot</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_tg">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>240</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Myanmar Text</family>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_tg_2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>420</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Myanmar Text</family>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>checkBox_tg_2</zorder>
     <zorder>checkBox_tg</zorder>
     <zorder>plot_totalErrors</zorder>
     <zorder>label_Solutions_4</zorder>
     <zorder>checkBox_ImpeulerTe</zorder>
     <zorder>checkBox_EulerTe</zorder>
     <zorder>checkBox_RkTe</zorder>
     <zorder>lineEdit_n0</zorder>
     <zorder>label_N_2</zorder>
     <zorder>label_n0</zorder>
     <zorder>lineEdit_N_2</zorder>
     <zorder>plot_totalErrors_2</zorder>
     <zorder>checkBox_RkTe_2</zorder>
     <zorder>checkBox_EulerTe_2</zorder>
     <zorder>label_Solutions_5</zorder>
     <zorder>checkBox_ImpeulerTe_2</zorder>
     <zorder>plot_button4_3</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QGraphicsView</extends>
   <header>qcustomplot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
